ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master__nto.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- dockerfile_path: build/Dockerfile
+  from: os
+  to: ci-artifacts
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.8"
```

Comment:
I think you're running twice (master & release-4.8) on OCP 4.8
for GPU Operator I have no "nightly" on master, only presubmit